### PR TITLE
Add test coverage to NodeJS GAPIC clients.

### PR DESCRIPTION
**Pull request by shinfan**
Refreshed and added test coverage (mocha unit test support) to the following APIs:
- Errorreporting
- Language
- Logging
- Monitoring
- Pubsub
- Speech
- Trace
- Vision

All tests have passed.

How-to-run:
Under the corresponding API directory, run `npm install` & `npm test`

cc @bjwatson @garrettjonesgoogle

### CODE REVIEW

File: generated/nodejs/google-cloud-node/packages/errorreporting/package.json [hunk 1]
```diff
@@ -3,19 +3,25 @@
...
   "author": "Google Inc.",
-  "description": "Stackdriver Clouderrorreporting library for Node.js",
+  "description": "Errorreporting client for Node.js",
   "main": "src/index.js",
   "files": [
     "src"
   ],
   "dependencies": {
-    "google-proto-files": "^0.7.0",
+    "google-proto-files": "^0.8.3",
```
Comment (jmuk) [inline generated/nodejs/google-cloud-node/packages/errorreporting/package.json]: The latest version of google-proto-files is 0.8.6

File: generated/nodejs/google-cloud-node/packages/errorreporting/package.json [hunk 2]
```diff
@@ -3,19 +3,25 @@
...
     "google-gax": "^0.7.0",
```
Comment (jmuk) [inline generated/nodejs/google-cloud-node/packages/errorreporting/package.json]: Use google-gax version `0.10.5` (or `0.10.6`?  @landrito -- are you gonna republish a newer version?)

Comment (landrito) [inline generated/nodejs/google-cloud-node/packages/errorreporting/package.json]: Use `0.10.6`. It was just published!

File: generated/nodejs/google-cloud-node/packages/errorreporting/package.json [hunk 3]
```diff
@@ -3,19 +3,25 @@
   "name": "@google-cloud/errorreporting",
   "version": "0.7.1",
   "author": "Google Inc.",
-  "description": "Stackdriver Clouderrorreporting library for Node.js",
+  "description": "Errorreporting client for Node.js",
```
Comment (shinfan) [inline generated/nodejs/google-cloud-node/packages/errorreporting/package.json]: This file is now auto generated and Errorreporting is the API name from the API config. If we need to add the Stackdriver we probably need to modify the api config file.

File: generated/nodejs/google-cloud-node/packages/errorreporting/package.json (hunk 2 above)
Comment (shinfan) [inline generated/nodejs/google-cloud-node/packages/errorreporting/package.json]: Done.

**Comment by shinfan** [comment]
Update: The `gax.StreamType.BIDI_STREAMING` issue is now resolved by updating the gax version to 0.10.6.

@jmuk @landrito PTAL

File: generated/nodejs/google-cloud-node/packages/errorreporting/src/v1beta1/index.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
-/*!
- * Copyright 2016 Google Inc. All Rights Reserved.
+/*
+ * Copyright 2016 Google Inc. All rights reserved.
```
Comment (jmuk) [inline generated/nodejs/google-cloud-node/packages/errorreporting/src/v1beta1/index.js]: This is funny, I didn't realized there was difference among capital R and r in our templates 😉 

But, it seems like, hand-written code in `google-cloud-node` uses capital R while GAPIC code uses small r. Tiny, but probably we want to update the template to use capital R.

**Pull request merged**